myapp/app/views/tasks/_form.html.erb

```diff
@@ -34,11 +34,12 @@
     <%= form.label :status, t('task.table_headers.status') %>
     <%= form.select :status, ["Not Started", "In Progress", "Done"] %>
   </div>
-
-  <div class="field">
+  
+  <%# todo: add labels to tasks %>
+  <%# <div class="field">
     <%= form.label :label_ids, t('task.table_headers.labels') %>
-    <%= form.collection_select :label_ids, @labels, :id, :name, {}, multiple: true %>
-  </div>
+    <%# <%= form.collection_select :label_ids, @labels, :id, :name, {}, multiple: true %> <%# %>
+  <%# </div> %> <%# %>
```

Comment:
sorry, what was this that you commented out? @@

Reply:
This part is for labels function, I will add it in the future. It is better to remove those in current step?

Reply:
it's fine, just for this kind of things, you can add a comment nearby so we know what it is and why that you want to comment it out for now.

Reply:
Sure, I have a todo comment.